packages/dx-react-demos/src/bootstrap3/data-types/editors.jsx

```diff
@@ -84,7 +90,11 @@ export default class Demo extends React.PureComponent {
         columns={columns}
         getRowId={getRowId}
       >
-        <BooleanTypeProvider />
+        <DataTypeProvider
```

Comment:
Why did you inline the setup of DataTypeProvider here instead of extracting it in a separate component?